user_guide_src/source/database/query_builder.rst

```diff
@@ -210,6 +210,28 @@ Permits you to write the FROM portion of your query::
     in the ``$db->table()`` function. Additional calls to ``from()`` will add more tables
     to the FROM portion of your query.
 
+**$builder->fromSubquery()**
+
+Permits you to write part of a FROM query as a subquery.
+
+This is where we add a subquery to an existing table.::
+
+    $subquery = $db->table('users');
+    $builder  = $db->table('jobs')->fromSubquery($subquery, 'alias');
+    $query = $builder->get();
+
+    // Produces: SELECT * FROM `jobs`, (SELECT * FROM `users`) AS alias
+
+Use the ``$db->newQuery()`` method to make a subquery the main table.::
+
+    $subquery = $db->table('users')->select('id, name');
+    $builder  = $db->newQuery()->fromSubquery($subquery, 't');
+    $query = $builder->get();
+
+    // Produces: SELECT * FROM (SELECT `id`, `name` FROM users) AS t
+
+.. note:: Only one subquery can be passed to a method.
```

Comment:
@iRedds I've already merged this, but I noticed when seeing the user guide.
Is the note necessary?

Reply:
Oops, I really missed that.